package.json

```
@@ -11,6 +11,14 @@
     "prepublishOnly": "cp ./src/components/Clamp.js .",
     "publish": "rm ./Clamp.js"
   },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Justineo/vue-clamp.git"
+  },
+  "bugs": {
+    "url": "https://github.com/Justineo/vue-clamp/issues"
+  },
+  "homepage": "https://github.com/Justineo/vue-clamp#readme",
```

```suggestion
  "homepage": "https://justineo.github.io/vue-clamp/demo/",
```

Thanks! Let's make the demo site our homepage 😃